toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -84,6 +84,11 @@ func doCustomizations(buildDir string, baseConfigPath string, config *imagecusto
 		return err
 	}
 
+	err = enableVerityPartition(config.SystemConfig.Verity, imageChroot)
```

Comment:
I assume that dm-verity (like encryption) requires the boot partition to be separate from the rootfs partition. If this is the case, then a check should be added to make sure that is actually the case.